apps/src/templates/progress/FocusAreaIndicator.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * A component that adds a sort of ribbon looking focus indicator to the side
+ * of your row. Note: This is English only.
```

Comment:
Heh. I think the hesitancy came from the fact that it looks somewhat less ribbony than the spec ;-)